### DISCUSSION TITLE
Dead link on download.eclipse.org

### DISCUSSION THREAD

**Discussion by PatrickTasse** [eclipse-orbit/orbit-simrel#26] [Q&A] [answered]
Just a heads up, the link to [Orbit Restructuring Plan](https://github.com/orgs/eclipse-orbit/discussions/49) on https://download.eclipse.org/tools/orbit/downloads/ returns a 404.

**Comment by merks** [comment]
@jonahgraham 

I think when I archived the old orbit repo and made orbit-simrel's link on this pages the organization's default discussion:

https://github.com/eclipse-orbit/orbit-simrel

that the link got broken and that one needs to link here:

https://github.com/eclipse-orbit/orbit/discussions/49

Though at this point in time, it probably better to link to this:

https://github.com/orgs/eclipse-orbit/discussions/20

because that's current and contains the link to the older restructuring plan.

@PatrickTasse Thanks for reporting the issue!

**Reply by jonahgraham** [reply]
:-( Those files are managed on the now archived git repo by editing https://github.com/eclipse-orbit/orbit/blob/main/releng/website/downloads/index.php and running https://ci.eclipse.org/orbit/job/orbit-sync-html-files/.

**Reply by merks** [reply]
So you would need to set this to false temporarily in order to fix it:

https://github.com/eclipse-orbit/.eclipsefdn/blob/16564af69bf20ee18e460c6a2501baf821c85680/otterdog/eclipse-orbit.jsonnet#L57

That sounds like a bit of a pita... :scream:

**Reply by jonahgraham** [reply]
Probably best to move that directory to a new git repo - it was only added to that repo relatively recently as it used to be edited by ssh'ing into download.e.o.

**Reply by jonahgraham** [reply]
I can do it at some point - but not today.

**Reply by merks** [reply]
Yes, definitely not urgent!

**Comment by yanntm** [comment]
Hi, 
after much searching I end up here, but this *is* a bit urgent. 2023/12 has shipped but the info on how to migrate from R builds to the new organization does need to be put up and linked from the eclipse pages on orbit.
thank you.

**Reply by merks** [reply]
Is there missing information in this section on the main organization page?

https://github.com/eclipse-orbit#update-sites

As suggested there, the following is the primary place from which to get current Orbit content:

https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/

This documentation describes the general structure of such an update site:

https://eclipse.dev/justj/?page=tools#p2-anatomy

If there is more you need to know, I can try to help.

**Reply by yanntm** [reply]
Hi thanks for the answer.

It's more that what Google linked me to was this page :
https://github.com/eclipse-orbit/orbit
And of course the historic page on Eclipse site : 
https://download.eclipse.org/tools/orbit/downloads/

Both of them contain the same annoying 404 link to discussion #49.

They should both point to your very nice link : https://github.com/eclipse-orbit#update-sites

I usually go to orbit to get my R200XXXXX url every year or so when I update my eclipse dependencies. 

I'd have been fine with the new update site url, it's actually better than the old one because it honors the 2023-12 simrel name in its url, meaning less maintenance of my pom (which had both orbit and main eclipse release aligned).

However I kind of hit a wall of these 404 errors, and unfortunately I had to delve pretty deep into "issues" to reach the right link.

** 

So I believe that the sentence : Please see the [Orbit Restructuring Plan](https://github.com/orgs/eclipse-orbit/discussions/49) for details on how and where to consume Orbit bundles going forward. This page provides access to the historical builds based on Eclipse Bundle Recipes.

Is a really annoying and non informative sentence.

I suggest 
"Starting from 2023-09 eclipse simrel, the Orbit project has moved to Github, and the newer builds are available from the https://github.com/eclipse-orbit#update-sites update site."

Or something similar.

**Reply by merks** [reply]
I see, yes, when we moved the org discussions from the old repo to the new repo the links broke.  I'd have to un-archive the old repo it to change anything and then re-archive it.  There's just so much to do.  😱

**Comment by wimjongman** [comment]
The link is still dead.

**Reply by merks** [reply, marked as answer]
Rather than jump through hoops to promote content from an archived repository, I just modified the file with a simple sed script.

![image](https://github.com/eclipse-orbit/orbit-simrel/assets/208716/43225160-1f31-4dc2-a05b-4875a97e26af)
